types/workgrid-toolbar/index.d.ts

```diff
@@ -14,12 +14,15 @@ export interface WorkgridToolbarOptions {
     // Optional
     confirmUser?: boolean | false;
     debug?: boolean | false;
+    deferDisplay?: boolean | false;
     htmlDisplay?: string | 'block';
```

Comment:
Same here string | 'blocl' = string so just string here is needed
Please remove the redundant types in the file.